test/utils.js

```diff
@@ -206,9 +223,9 @@ module.exports.closePanel = async(driver) => {
 // first element is most recent ping
 // as seen in shield-study-addon-util's `utils.jsm`
 module.exports.getMostRecentPingsByType = async(driver, type) =>
-  driver.executeAsyncScript(async() => {
-    const typeArg = arguments[0];
-    const callback = arguments[arguments.length - 1];
```

Comment:
Yes, this works! I had tried to get this to work previously, but I think I had the order switched (ie. `async(callback, typeArg)`) so I decided to use `...args` instead.